en/docs/assets/attachments/install-and-setup/apim210_to_apim320_gateway_artifact_migrator.sh

```diff
@@ -16,7 +16,9 @@ echo 'starting gateway artifact migration...'
 
 find . -wholename './[0-9]*/synapse-configs/default/*.xml' -print0 -o -name '*.xml' -print0 | xargs -0 perl -i -pe's/org.wso2.carbon.mediator.cache.digest.DOMHASHGenerator/org.wso2.carbon.mediator.cache.digest.REQUESTHASHGenerator/'
 find . -wholename './[0-9]*/synapse-configs/default/*.xml' -print0 -o -name '*.xml' -print0 | xargs -0 perl -i -pe's/org.wso2.caching.digest.REQUESTHASHGenerator/org.wso2.carbon.mediator.cache.digest.REQUESTHASHGenerator/'
-find . -wholename './[0-9]*/synapse-configs/default/*.xml' -print0 -o -name '*.xml' -print0 | xargs -0 perl -i -pe's/<handler class="org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler"\/>/<handler class="org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler">\n\t\t\t\t<property name="RemoveOAuthHeadersFromOutMessage" value="true"\/>\n\t\t\t<\/handler>/'
+find . -wholename './[0-9]*/synapse-configs/default/*.xml' -print0 -o -name '*.xml' -print0 | xargs -0 perl -i -pe's/<handler class="org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler"\/>/<handler class="org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler">\n\t\t\t\t<property name="RemoveOAuthHeadersFromOutMessage" value="true"\/>\n\t\t\t<\/handler>/'find . -wholename './[0-9]*/synapse-configs/default/*.xml' -print0 -o -name '*.xml' -print0 | xargs -0 perl -pi -ne 'BEGIN {undef $/} s/<handler class="org.wso2.carbon.apimgt.gateway.handlers.throttling.APIThrottleHandler">(.|\n)*?<\/handler>/<handler class="org.wso2.carbon.apimgt.gateway.handlers.throttling.ThrottleHandler"\/>/'
```

Comment:
`find . -wholename './[0-9]*/synapse-configs/default/*.xml' -print0 -o -name '*.xml' -print0 | xargs -0 perl -i -pe's/<handler class="org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler"\/>/<handler class="org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler">\n\t\t\t\t<property name="RemoveOAuthHeadersFromOutMessage" value="true"\/>\n\t\t\t<\/handler>/'find . -wholename './[0-9]*/synapse-configs/default/*.xml' -print0 -o -name '*.xml' -print0 | xargs -0 perl -pi -ne 'BEGIN {undef $/} s/<handler class="org.wso2.carbon.apimgt.gateway.handlers.throttling.APIThrottleHandler">(.|\n)*?<\/handler>/<handler class="org.wso2.carbon.apimgt.gateway.handlers.throttling.ThrottleHandler"\/>/'
Write`

seems there's a typo here (2 find statements are there)

Reply:
@msm1992 , this has been fixed in 94862ca9357463ca7caa5756fff0278ab3e10305 
Please check and PRAM